### DISCUSSION TITLE
Contribution to the development of the Mono v2 Runtime

### DISCUSSION THREAD

**Discussion by paradoxcyx** [citizenfx/fivem#2311] [Engineering Discussion]
Hi All,

I would like to contribute to the Mono v2 Runtime if at all possible. I am not sure if people outside the Cfx dev team is "allowed" to contribute to it..

I am new here, so this might be really vague.

I have tons of C# experience, but not so much on C++.

Please let me know :)

**Comment by Sasino97** [comment]
Almost sure Mono v2 was discontinued ...

**Reply by paradoxcyx** [reply]
Who and where does it say this?
The pilot date for Mono V2 is still being bumped though.
